tools/topology/topology2/include/common/common_definitions.conf

```diff
@@ -1,4 +1,4 @@
-# Commonly used definitions in cavs topologies
+# Commonly used definitions in topologies
```

Comment:
We'll need to figure out how tp move the SSP_BLOB version definitions to some Intel specific file, but rest are indeed generic!